analysis/src/analyzer.js

```diff
@@ -61,6 +61,17 @@ class AnalyzerRunner {
       const rootUrl = await analyzer.urlResolver.resolve('');
       const isInPackage = feature => feature.sourceRange &&
         feature.sourceRange.file.startsWith(rootUrl);
+      // Filter out top level test/ and demo/ directories.
+      const isTest = (feature) => {
+        if (!feature.sourceRange) {
+          return false;
+        }
+        const relativePath = feature.sourceRange.file.substring(rootUrl.length);
+        if (relativePath.startsWith('test/') || relativePath.startsWith('demo/')) {
```

Comment:
maybe "tests?/"

Reply:
Not a regex, but I added a check for `tests/`.

Reply:
lgtm